third_party/hal_sifli/cconfig.h

```diff
@@ -0,0 +1,21 @@
+#ifndef CCONFIG_H__
+#define CCONFIG_H__
+/* Automatically generated file; DO NOT EDIT. */

```

Comment:
How is this generated? Until it's automatically generated in our build, add a note as to how it can be re-generated?

Reply:
This was initially generated in Sifli SDK, however, manually adjusted as we do not use their Kconfig system. It's not gonna be changed, so maybe I can just delete this comment as it's not entirely true anymore.